SU-HSDO/stanford_mrc
# MRCD8-311 visitors grid/list floated styles

# READY FOR REVIEW

# Summary
- Visitors Grid is now stacked cards and appears next to events on the program/event series pages, so:
- Visitor images are now floated to the left

# Needed By (Date)
- End of sprint (3/29)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Test an even series pages with visitors
3. Ensure that visitor images are floated to the left

# Affected Projects or Products
- MRC
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-311

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
